lightning/config/config.go

```diff
@@ -61,7 +64,10 @@ const (
 	ErrorOnDup = "error"
 )
 
-var defaultConfigPaths = []string{"tidb-lightning.toml", "conf/tidb-lightning.toml"}
+var (
+	defaultConfigPaths    = []string{"tidb-lightning.toml", "conf/tidb-lightning.toml"}
+	supportedStorageTypes = []string{"file", "local", "s3"}
```

Comment:
what's the different between file and local?

Reply:
I think there is no difference, `storage.ParseBackend` just treat them the same way